### PR TITLE
Set runtime_include_dir in Paddle.__init__.py

### PR DESCRIPTION
**Pull request by zhhsplendid**
### PR types

New features
### PR changes

Others
### Describe

Paddle don't have to set runtime_include_dir during run CINN.

### CODE REVIEW

File: paddle/fluid/pybind/pybind.cc [hunk 1]
```diff
@@ -207,6 +207,14 @@ bool IsCompiledWithMKLDNN() {
 #endif
 }
 
+bool IsCompiledWithCINN() {
```
Comment (thisjiang) [inline paddle/fluid/pybind/pybind.cc:210]: 感觉返回值可以加`constexpr`，不过加不加都一样

**Review by thisjiang — APPROVED** [review]
LGTM

File: paddle/fluid/pybind/pybind.cc (hunk 1 above)
Comment (zhhsplendid) [reply paddle/fluid/pybind/pybind.cc:210]: 谢谢，跟别的地方保持一致，先不加吧

**Review by lanxianghit — APPROVED** [review]

**Review by Xreki — APPROVED** [review]
LGTM for the change of framework.py

**Review by dingjiaweiww — APPROVED** [review]

**Review by Ligoml — APPROVED** [review]
image
LGTM

**Pull request merged**